opentreemap/treemap/js/src/lib/urlState.js

```diff
@@ -185,6 +195,13 @@ module.exports = {
         });
```

Comment:
Looks like there is already an open issue for this https://github.com/OpenTreeMap/otm-core/issues/2784